src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.ILLink.targets

```diff
@@ -22,13 +22,18 @@ This file contains the .NET 5-specific targets to customize ILLink
     <UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == ''">true</UseSystemResourceKeys>
     <HttpActivityPropagationSupport Condition="'$(HttpActivityPropagationSupport)' == ''">false</HttpActivityPropagationSupport>
     <StartupHookSupport Condition="'$(StartupHookSupport)' == ''">false</StartupHookSupport>
+    <XATypeRegistrationFallback Condition="'$(XATypeRegistrationFallback)' == ''">false</XATypeRegistrationFallback>
   </PropertyGroup>
 
   <Target Name="_PrepareLinking"
       Condition=" '$(PublishTrimmed)' == 'true' "
       AfterTargets="ComputeResolvedFilesToPublishList"
       DependsOnTargets="GetReferenceAssemblyPaths;_CreatePropertiesCache">
     <ItemGroup>
+      <RuntimeHostConfigurationOption Include="Java.Interop.TypeManager.TypeRegistrationFallbackIsEnabled"
+          Condition="'$(XATypeRegistrationFallback)' != ''"
+          Value="$(XATypeRegistrationFallback)"
+          Trim="true" />
```

Comment:
Where is this value actually stored? Is it a `*.runtimeconfig.json` file?

I don't see how that would get loaded at runtime, as we didn't do anything to put it in the `.apk` -- I don't think the runtime would know how to load it from the `.apk` either.

Reply:
The feature switch value can be indeed set in a `runtimeconfig.json` file if I understand it correctly. In our case we care only about the linker, where it is passed as `--feature Java.Interop.TypeManager.TypeRegistrationFallbackIsEnabled false` option during the build.